pup-0005.md

```diff
@@ -0,0 +1,148 @@
+on to this
+[statement](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/process/kernel-enforcement-statement.rst).
+
+In November 2017, a coalition of companies led by Red Hat
+[announced](https://www.redhat.com/en/about/press-releases/technology-industry-leaders-join-forces-increase-predictability-open-source-licensing)
+that they were adopting a similar commitment, the CCRC, for all of
+their copyrighted code that was licensed under GPLv2, LGPLv2.0 and
+LGPLv2.1. Red Hat persuaded an additional set of companies to
+[adopt](https://www.redhat.com/en/about/press-releases/momentum-builds-new-wave-technology-industry-leaders-join-efforts-increase-predictability-open-source-licensing)
+the CCRC in March 2018.
+
+Red Hat would like to encourage projects to adopt the CCRC as
+well. The issue is relevant for projects maintained or led by Red Hat
+developers for a few reasons: (1) GPLv2 and LGPLv2.x are licenses that
+many Red Hat developers continue to prefer; (2) projects led by Red
+Hat developers accept contributions under the project license (as
+opposed to an asymmetrical contributor license agreement); and (3)
+traditionally Red Hat developers have exercised broad discretion on
+project licensing policy.
+
+Pulp is a good candidate for adoption of the CCRC at the project
+level. Pulp is a GPLv2-licensed community project of commercial
+significance to Red Hat, and Pulp has been making significant efforts
+to build a community around the project, which over time will
+encourage contributions to the project from various copyright holders.
+
+## Common Cure Rights Commitment, version 1.0
+
+Common Cure Rights Commitment, version 1.0
+ 
+Before filing or continuing to prosecute any legal proceeding or claim
+(other than a Defensive Action) arising from termination of a Covered
+License, we commit to extend to the person or entity ('you') accused
+of violating the Covered License the following provisions regarding
+cure and reinstatement, taken from GPL version 3. As used here, the
+term 'this License' refers to the specific Covered License being
+enforced.
+
+    However, if you cease all violation of this License, then your
+    license from a particular copyright holder is reinstated (a)
+    provisionally, unless and until the copyright holder explicitly
+    and finally terminates your license, and (b) permanently, if the
+    copyright holder fails to notify you of the violation by some
+    reasonable means prior to 60 days after the cessation.
+
+    Moreover, your license from a particular copyright holder is
+    reinstated permanently if the copyright holder notifies you of the
+    violation by some reasonable means, this is the first time you
+    have received notice of violation of this License (for any work)
+    from that copyright holder, and you cure the violation prior to 30
+    days after your receipt of the notice.
+
+We intend this Commitment to be irrevocable, and binding and
```

Comment:
don't we want to specify some time here? proposer can not be bound indefinitely and any kind of revocation is without the effect in case there will be the need of change/modification

Reply:
I missed this comment in my review. I'm confused at what specifying time here would intend to do. Also who is the proposer here? I overall don't understand this comment.

Reply:
proposer is legal term, usually it;s a party which formally suggests a plan, law, legislation etc. [0]
In this specific case - we are proposer.

w/r to 'irrevocable right/agreement/contract/etc' ( also a legal term) by default it is something that cannot be altered, withdrawn or terminated and if no date or period is specified of its' validity then you are bound to this indefinitely.
[0] https://www.ldoceonline.com/dictionary/proposer

i thought maybe @richardfontana could answer this question, maybe i am wring since i do not remember all the clauses.

Reply:
@ipanova - I read that as our intentions are that "we mean it. we're not going to randomly back out of our commitment and give you no opportunity to cure the violations."